server/schedulers/hot_test.go

```diff
@@ -52,85 +53,135 @@ func (s *testHotWriteRegionSchedulerSuite) TestSchedule(c *C) {
 	tc.AddLabelsStore(7, 0, map[string]string{"zone": "z5", "host": "h7"})
 	tc.SetStoreDown(7)
 
-	// Report store written bytes.
-	tc.UpdateStorageWrittenBytes(1, 75*1024*1024)
-	tc.UpdateStorageWrittenBytes(2, 45*1024*1024)
-	tc.UpdateStorageWrittenBytes(3, 45*1024*1024)
-	tc.UpdateStorageWrittenBytes(4, 60*1024*1024)
+	//| store_id | write_bytes_rate |
+	//|----------|------------------|
+	//|    1     |       7.5MB      |
+	//|    2     |       4.5MB      |
+	//|    3     |       4.5MB      |
+	//|    4     |        6MB       |
+	//|    5     |        0MB       |
+	//|    6     |        0MB       |
+	tc.UpdateStorageWrittenBytes(1, 7.5*MB*statistics.StoreHeartBeatReportInterval)
```

Comment:
`statistics.StoreHeartBeatReportInterval` is too long.

Reply:
I noticed that, but I didn't find any better way to solve it.